new-site/src/docs/elements/components/card/code.mdx

```diff
@@ -26,29 +23,39 @@ import Playground from '../../../../components/Playground';
 ```
 
 ```html
-<div>
-  <div class="hds-card"></div>
-  <div class="hds-card hds-card--border"></div>
-</div>
+  <div class="hds-card">
```

Comment:
Also we should consider rendering the empty variant on different background color than white, so it would show something other than just white everywhere.

Reply:
I think we have similar examples in our current documentation

Reply:
I think we could go through the content separately.